few update

sobuj53 opened the pull request:
Sqlite db update and few other update to make Clipjump working

sobuj53 committed:
SqliteDB update

source https://github.com/AHK-just-me/Class_SQLiteDB/blob/master/Sources_v1.1/Class_SQLiteDB.ahk

sobuj53 committed:
icon update

icon updated

sobuj53 committed:
Clipjump update

updated to change folder 📂 path `\` to `/` for using in R

sobuj53 committed:
sqlite dll update

Sqlite dll update to latest version

sobuj53 committed:
plugin update

Updated to solve issue https://github.com/aviaryan/Clipjump/issues/134

Skrell commented:
Thank you so much for keeping the best clipboard manager on the internet working!!! <3  How do I generate a new Clipjump.exe to use with your source code?

sobuj53 committed:
redundant

not needed

Skrell commented:
When I try running your forked code with the original Clipjump.exe I get an error saying it can't find the sqlite3.dll which I can clearly see is present?

sobuj53 committed:
RunAs*

As Clipjump asks for Run as Admin, that's why added Runas Admin method to ease the workload on user.

sobuj53 committed:
Added force close

For some reason Exiting Clipjump from tray doesn't properly close the process. That's why added Nircmd force close method to completely close on exit. But this requires Nircmd to be installed in, `%USERPROFILE%\AppData\Local\Microsoft\WindowsApps`. Just download Nircmd from https://www.nirsoft.net/utils/nircmd-x64.zip and extract to the mentioned folder.

sobuj53 commented:
Hi thank you for your interest. You're welcome to fork my repo or download the content as zip. Clipjump.exe won't work in this case because I didn't create executable file with the latest changes so it'll throw out error. Just use `Clipjump_x64.ahk`, it'll do the job. You can create a task schedule to automate this process. If you want to create task schedule then do select `Run with highest privileges` in general tab, set triggers at `Log on` at actions tab start a program, `Program/Script` value = `"C:\Program Files\AutoHotkey\AutoHotkey.exe"` arguments value = `"<path>\Clipjump_x64.ahk"` of course change `<path>` to your Clipjump directory.

Skrell commented:
Would you mind making a new Clipjump.exe  ?

sobuj53 commented:
> Would you mind making a new Clipjump.exe ?

nope, as it's basically the same thing and also it gives me freedom to change or update the code without hassle.

sobuj53 committed:
Release update.

Added Release update and Nircmd Dependency explanation.

Skrell commented:
Ok I did what you suggested and it appeared to work fine, except for 2 things:  
1) in Task explorer the process shows up as "Autohotkey Unicode 64-bit" instead of "Clipjump"
2) When I attempt to exit clipjump from the tray menu I get the error "Nircmd not found!  Download Nircmdm from, https://www.nirsoft.net/utils/nircmd.html" ??

sobuj53 commented:
1. I can't do anything about that, it’s by design.
2. `Nircmd` is required for this build read, https://github.com/sobuj53/Clipjump

Skrell commented:
Why is nircmd required and why does it have to be in that directory?  Can you change your scripts to at least use nircmd from the directory I run clipjump.ahk in?

sobuj53 commented:
I specifically wrote why it’s required in the following commit https://github.com/sobuj53/Clipjump/commit/a74b30376ad3ae35a0e9b93576c67e020e6427df 
In short it’s not possible or maybe possible, you have source code so you can always modify it to your need. I saw it fits my requirement so I adopted it.

sobuj53 committed:
Removed Nircmd Dependency

Removed External Nircmd dependency and replaced with simple function.

sobuj53 committed:
Removed Nircmd dependency

Removed Nircmd dependency and replaced with simple ahk built in function.

sobuj53 committed:
SQLite.dll update

Updated SQLite.dll to 3.35.4
